data/emmyVon.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "emmyVon",
+  "bio": "FrontEnd Developer | React Specialist | Newbie | Looking to make strong ground is the Tech space , very open and willing to learning",
+  "links": [
+    {"group":"Twitter",
+    "name":"emmyDayvon",
+    "Url":"https://twitter.com/emmyobiezuo",
+    "icon":"FaSquareXTwitter"},
+     {"group":"Linkedin",
+    "name":"Emmanuel Obiezuo",
+    "Url":"https://www.linkedin.com/in/emmanuel-obiezuo-a0018a279/",
+    "icon":"FaLinkedin"},
```

```suggestion
    "icon":"FaLinkedin"}
```
The trailing comma isnot required